packages/pipeline/src/TooltipText.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ChevronRight from '@leafygreen-ui/icon/dist/ChevronRight';
+
+import { tooltipTextStyles } from './styles';
+import { palette } from '@leafygreen-ui/palette';
+
+export interface TooltipTextProps {
+  hiddenStages: Array<string | null>;
+}
+
+/**
+ * # TooltipText
+ *
+ * React Component to render the TooltipText for the number of hidden hiddenStages in the Pipeline component.
+ *
+ * ```
+ * <TooltipText />
+ * ```
+ * @internal
+ */
+const TooltipText = ({ hiddenStages }: TooltipTextProps) => {
+  const tooltipChildren = [];
+
+  if (hiddenStages.length > 0) {
+    for (let i = 0; i < hiddenStages.length; i++) {
```

Comment:
Minor - Is `if (hiddenStages.length > 0)` necessary here? I assume it would just be caught by the loop guard.